src/Symfony/Component/Translation/Translator.php

```diff
@@ -155,6 +165,19 @@ public function getFallbackLocales(): array
         return $this->fallbackLocales;
     }
 
+    public function addGlobalParameter(string $id, string|int|float|TranslatableInterface $value): void
+    {
+        $this->globalParameters[$id] = $value;
+    }
+
+    /**
+     * @internal
```

Comment:
let's remove this annotation? I don't see it helping us (maintainers) much, while it might be useful for userland, next to addGlobalParameter

Reply:
Sure ! It's done.

Is there any reason to keep it for the `getFallbackLocales` method ?